Content.Server/Nutrition/EntitySystems/DrinkSystem.cs

```diff
@@ -184,40 +178,31 @@ private void OnUse(EntityUid uid, DrinkComponent component, UseInHandEvent args)
         if (args.Handled)
             return;
 
-        if (!component.Opened)
-        {
-            //Do the opening stuff like playing the sounds.
-            _audio.PlayPvs(_audio.GetSound(component.OpenSounds), args.User);
-
-            SetOpen(uid, true, component);
-            return;
-        }
-
         args.Handled = TryDrink(args.User, args.User, component, uid);
     }
 
-    private void HandleLand(EntityUid uid, DrinkComponent component, ref LandEvent args)
+    private void OnPressurizedDrinkLand(EntityUid uid, PressurizedDrinkComponent comp, ref LandEvent args)
     {
-        if (component.Pressurized &&
-            !component.Opened &&
-            _random.Prob(0.25f) &&
-            _solutionContainer.TryGetSolution(uid, component.SolutionName, out var interactions))
+        if (!TryComp<DrinkComponent>(uid, out var drink) || !TryComp<OpenableComponent>(uid, out var openable))
+            return;
+
+        if (!openable.Opened &&
```

Comment:
WTF???? how does this not blow up the build cuz of access????

Reply:
everything can read fields